homeassistant/components/blastbot_cloud/config_flow.py

```diff
@@ -0,0 +1,63 @@
+"""Blastbot Cloud config flow."""
+
+from aiohttp import ClientError
+from blastbot_cloud_api.api import BlastbotCloudAPI
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.core import callback
+
+from .const import DOMAIN, LOGGER
+
+
+class BlastbotCloudConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Blastbot Cloud config flow."""
+
+    def __init__(self):
+        """Initialize."""
+        self.data_schema = {
+            vol.Required(CONF_USERNAME): str,
+            vol.Required(CONF_PASSWORD): str,
+        }
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+
+        if not user_input:
```

Comment:
Invert this check and put the show form at the end of the step with possible errors.